cpp/src/groupby/hash/single_pass_functors.cuh

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2020-2024, NVIDIA CORPORATION.
+ * Copyright (c) 2024, NVIDIA CORPORATION.
```

Comment:
Is it intentional to remove the years? This is an existing file, right?

Reply:
Fixed. Thanks!